src/web/app/pages-instructor/instructor-course-details-page/instructor-course-details-page.component.html

```diff
@@ -137,7 +137,7 @@ <h4 class="modal-title">Confirm Deletion</h4>
     <div class="modal-body">
       <tm-ajax-preload *ngIf="loading"></tm-ajax-preload>
       <div [innerHTML]="convertToHtmlTable(courseStudentListAsCsv)"></div>
-      <b>{{ isAjaxSuccess ? '' : 'Failed to load. Please reopen modal.' }}</b>
+      <b>{{ isAjaxSuccess ? '' : 'There was an error Loading this file. Please try again.' }}</b>
```

Comment:
Minor nit, but seeing that the entire issue is about crafting the message, the `L` in `Loading` shouldn't be capitalised. The text is otherwise fine, so just fix that bit :)

Reply:
Thanks, I just changed it now